ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.1.yaml

```diff
@@ -14,11 +14,11 @@ base_images:
   tests-private-postupg:
     name: tests-private
     namespace: ci
-    tag: "4.13"
+    tag: "4.12"
```

Comment:
@liangxia postupg should be 4.13. it is executed after upgraded to 4.13.

same comment for other code in this PR.

Reply:
Good catch. Misread with the cucushift one.